packages/web-components/src/horizontalbarchart/horizontalbarchart.stories.ts

```diff
@@ -189,17 +189,16 @@ export default {
 
 export const Basic: Story<FluentHorizontalBarChart> = renderComponent(storyTemplate).bind({});
 
-export const HBCAbsoluteScale: Story<FluentHorizontalBarChart> = renderComponent(html<
-  StoryArgs<FluentHorizontalBarChart>
->`
+export const singleBarHBC: Story<FluentHorizontalBarChart> = renderComponent(html<StoryArgs<FluentHorizontalBarChart>>`
   <div>
-    <fluent-horizontalbarchart style="width: 100%" data="${JSON.stringify(hbcData)}"> </fluent-horizontalbarchart>
+    <fluent-horizontalbarchart style="width: 100%" data="${JSON.stringify(singleBarHBCData)}">
+    </fluent-horizontalbarchart>
   </div>
 `);
 
 export const HBCNMVariant: Story<FluentHorizontalBarChart> = renderComponent(html<StoryArgs<FluentHorizontalBarChart>>`
```

Comment:
update this name also.

Reply:
NMVariant, I am planning to take in separate PR

Reply:
What should I rename this as ?